src/lineChunked.js

```diff
@@ -818,8 +813,9 @@ export default function () {
     if (clipPath.empty()) {
       clipPath = defs.append('clipPath')
         .attr('class', className)
-        .attr('id', `d3-line-chunked-clip-${chunkName}-${counter++}`);
+        .attr('id', `d3-line-chunked-clip-${chunkName}-${counter}`);
     }
+    counter += 1;
```

Comment:
Why move the counter increment outside of the if statement?

Reply:
Good question, it's not that I have any preference personally but the eslint test enforces this. The tests fail otherwise.